libraries/hwlib/library/targets/hwlib-arduino-due.hpp

```diff
@@ -861,6 +895,172 @@ class pin_adc : public hwlib::adc {
    }
 };
 
+class usart_bus {
+private:
+//    Pio & tx_port;
+//    Pio & rx_port;
+//    uint32_t tx_mask;
+//    uint32_t rx_mask;
+
+    Usart * hardwareUSART = nullptr;
+    unsigned int baudrate = BMPTK_BAUDRATE;
+    int n;
+    bool USART_controller_initialized = false;
+
+    Queue<uint8_t, 250> input_buffer;
+   
+    inline uint8_t receive_byte(){
```

Comment:
Is there a specific reason to force the compiler to inline this? The same for all the other inlines as well?